addons/html_editor/static/src/others/embedded_components/plugins/video_plugin/video_plugin.js

```diff
@@ -1,73 +1,47 @@
 import { Plugin } from "@html_editor/plugin";
 import { _t } from "@web/core/l10n/translation";
-import { VideoSelectorDialog } from "@html_editor/others/embedded_components/plugins/video_plugin/video_selector_dialog/video_selector_dialog";
-import { renderToElement } from "@web/core/utils/render";
 import { isHtmlContentSupported } from "@html_editor/core/selection_plugin";
+import { EmbeddedVideoSelector } from "./video_selector_dialog/embedded_video_selector";
 
 export class VideoPlugin extends Plugin {
     static id = "video";
-    static dependencies = ["embeddedComponents", "dom", "selection", "link", "history"];
+    static dependencies = ["embeddedComponents", "media"];
+    static defaultConfig = {
+        allowEmbeddedVideo: true,
```

Comment:
This new "allowEmbeddedVideo" option serves a double-purpose:
- to be able to switch it off eventually (no video insertion as embedded component allowed)
- to communicate to the Media Dialog plugin that there is a video tab in it, so that the correct discription to the "/media" command can be generated.